report/sections/Conclusion/conclusion.tex

```diff
@@ -0,0 +1,21 @@
+The main goal of this project was to create an internet-based application that would solve some problem.
+Furthermore, another requirement in the study regulation is that the developed system also needs to be tested for scalability.
+In this project, we developed a system for lecturers of the \textit{Programming Paradigms} course that can provide fast and efficient feedback on student exercise submissions.
+The system allows lecturers to create exercises for students and define accompanying tests for verifying correctness of student solutions.
+
+To create this application, we developed a working user interface that students and lecturers can use in order to interact with the system.
+The user interface was developed with user-friendliness in mind, providing minimal hindrance in accessing and solving exercises.
+A backend was developed in order to process and serve frontend requests.
+Supplementing this, a database was also implemented to store information regarding users and exercises.
```

Comment:
```suggestion
```

Reply:
I dont like that we go from Test Runner -> one sentence about DB -> Test Runner. I have made a suggestion to get DB before.

Reply:
And we didn't implement a database